src/repository/mysql/resolution.js

```diff
@@ -8,13 +8,19 @@ export default class MySQLResolution {
   }
 
   async add(data) {
-    let expire = null;
+    const resolutionData = {
+      doctor_name: data.doctorName,
+      doctor_specialization: data.doctorSpecialization,
+      patient_id: data.patientId,
+      ttl: data.ttl,
+      resolution: data.resolution,
+    };
 
-    if (data.ttl > 0) {
-      expire = new Date(getExpiration(data.ttl));
-    }
+    const expire = resolutionData.ttl > 0
+      ? new Date(getExpiration(resolutionData.ttl))
+      : null;
 
-    return this.Resolution.create({ expire, ...data });
+    return this.Resolution.create({ expire, ...resolutionData });
```

Comment:
Instances of classes should be named from lower case char.

Reply:
:white_check_mark: Fixed